Update n1 waf docs and changelog to reference default log profiles an… (#2110)
nginx/documentation: content/nginx-one-console/changelog.md

Stay up-to-date with what's new and improved in the F5 NGINX One Console.

## July 16, 2026

### F5 WAF for NGINX: Built-In Log Profile Support

> F5 WAF for NGINX built-in log profiles are now available as starting points while creating new log profiles. The configuration editor also now supports auto-complete for built-in log profile names, enabling easier reference to them.

Reviewer: Can we rewrite this in active voice? For example: "You can now use built-in log profiles as starting points when creating new log profiles."

> You can now use F5 WAF for NGINX built-in log profiles as starting points while creating new log profiles. The configuration editor also now supports auto-complete for built-in log profile names, making them easier for you to reference within a config.

Reviewer 2: ```suggestion
You can now use built-in F5 WAF for NGINX log profiles as starting points when you create new log profiles. The configuration editor now supports autocomplete for built-in log profile names, making it easier to reference them in a configuration.
```

### F5 WAF for NGINX: Log Profile Copy Support

> Enabled making a copy of log profiles via the log profile list row actions so that existing log profiles can be easily extended without overwriting their existing content.

Reviewer: "Enabled making a copy of..." is awkward. Could we rewrite this as "You can now copy log profiles from the log profile list..."?

> You can now make a copy of log profiles via the log profile list row actions so that you can easily extend existing log profiles without overwriting their existing content.

Reviewer 2: ```suggestion
You can now copy log profiles from the log profile list. Use row actions to create a copy and extend an existing log profile without overwriting its content.
```
